### PR TITLE
DACT-1591: Update Director Details Page - English Localisation

### PR DESCRIPTION
**Pull request by leighmcclurg**
https://companieshouse.atlassian.net/browse/DACT-1591

- English / Welsh (not translated) files added
- Update to update and address details pages for localisation

### CODE REVIEW

File: locales/en/update-director-details.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "updateDirectorDetailsTitle" : "View and update the director's details",
+    "updateDirectorDetailsHint" : "If you need to tell us about more than one update, and they happened on different dates, you must submit them separately.",
+    "updateDirectorDetailsInsetText" : "Any updates you make to a director's details are pending until you’ve submitted this filing, and we’ve accepted it.",
+    "updateDirectorDetailsHeading" : "Director details",
+    "updateDirectorDetailsUpdate" : "Update",
+    "updateDirectorDetailsName" : "Name",
+    "updateDirectorDetailsNationality" : "Nationality",
+    "updateDirectorDetailsOccupation" : "Occupation",
+    "updateDirectorDetailsContinueButton" : "Continue",
```
Comment (jbestwickch) [inline locales/en/update-director-details.json:10]: The file furniture.json contains commonly used strings such as "Continue"

File: src/controllers/update/update.director.details.controller.ts [hunk 2]
```diff
@@ -19,6 +20,8 @@ export const get = async (req: Request, resp: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
     const officerFiling: OfficerFiling = await getOfficerFiling(session, transactionId, submissionId);
+    const locales = getLocalesService();
+    const lang = selectLang(req.query.lang);
     let updatedData = false;
```
Comment (olaych) [inline src/controllers/update/update.director.details.controller.ts:26]: Seems currentUrl is missing (needed for localisation toggle). Missed it out for active directors too

File: locales/en/update-director-details.json (hunk 1 above)
Comment (leighmcclurg) [reply locales/en/update-director-details.json]: Thanks, have updated this now

File: src/controllers/update/update.director.details.controller.ts (hunk 2 above)
Comment (leighmcclurg) [reply src/controllers/update/update.director.details.controller.ts:26]: Thanks, have added currentUrl to both update.director.details and active.directors

**Review by jbestwickch — APPROVED** [review]

**Review by clewis1 — APPROVED** [review]

**Pull request merged**